client/src/components/PinkHeader/PinkHeader.js

```diff
@@ -0,0 +1,102 @@
+
+import React, { Component } from 'react';
+import "./PinkHeader.css";
+import Search from "./search@2x.png"
+import '../../App.css';
+
+// Class Based React Component
+class Banner extends Component {
+  constructor(props) {
+    super(props);
+    console.log(props);
+
+    // Default CSS class to apply to the Component
+    this.state = {
+      classList: "PinkHeader"
+    };
+  }
+
+  render() {
```

Comment:
Can you find a way to write this that doesn't include so much duplication? We shouldn't need these massive if statements